services/registration/src/routes/statistics.ts

```diff
@@ -351,7 +353,8 @@ statisticsRouter.route("/").get(
         const numTotal = branchAggregates.reduce((acc, aggregate) => acc + aggregate.count, 0);
 
         confirmationBranchStatistics[branch.name] = {
-          confirmed: statusCount[StatusType.CONFIRMED] || 0,
+          confirmed:
+            (statusCount[StatusType.CONFIRMED] || 0) + (statusCount[StatusType.CHECKED_IN] || 0),
```

Comment:
Confirmed and checked-in are different statuses, we shouldn't group them together